### ISSUE TITLE
Danish app language with entity wit/location or wit/datetime gives error 500 on any message query

### ISSUE THREAD

**Issue by geekgarage** [wit-ai/wit#2176]
If you create an app with danish language. example 276405790986811
The second you add an entity either 'wit/location' and/or 'wit/datetime' It doesn't matter if you train any utterances with the entities it will give you code 500 on any message query. 
I've tested every single entity by adding them all one after one, and only wit/location and wit/datetime casuses problems for a danish app.

I tried german just to see if it was any non-English language but it works with wit/location and wit/datetime

**Comment by geekgarage** [comment]
So i tried again, doing two new apps one for english and one for danish. everything went fine when running  the english and training on utterances BUT, when doing the same on the app set to danish mid trying to configure a traning for the danish app, the UI resets out of no where and wants me to go through the tutorial again instead of showing me the utterances i wasn't finished configuring, it also applies what ever I've half configured, and then the response code is 500 if i try any GET Message query. But when i remove the intent and Entities from that half configured utterance i can again run GET Message and get a response 200 and try again

**Comment by geekgarage** [comment]
A video showing the problem with danish language. 
App IDs used in the video 
test_eng = 554716628906765
test_da = 276405790986811
Video have no audio as it's easy to see what I'm doing. 
https://youtu.be/bWbO2TuvSjM

**Comment by geekgarage** [comment]
I've found the issue.
If you create an app, at least with danish language.
The second you add an entity either 'wit/location' and/or 'wit/datetime' It doesn't matter if you train any utterances with the entities it will give you code 500 on any message query. 
I've tested every single entity by adding them all one after one, and only wit/location and wit/datetime casuses problems for a danish app.

I tried german just to see if it was any language but it works with wit/location and wit/datetime

Updated title to be more specific
